Firmware/Marlin_main.cpp

```diff
@@ -6102,7 +6105,7 @@ SERIAL_PROTOCOLPGM("\n\n");
     case 190: 
     #if defined(TEMP_BED_PIN) && TEMP_BED_PIN > -1
     {
-        bool CooldownNoWait = false;
+        CooldownNoWait = false;
```

Comment:
As written before, this would cause M190 to affect the wait for extruder temperature during recovery.. which is not good. I would keep this one as-is.